packages/composer/lib/proxy.js

```diff
@@ -5,9 +5,18 @@ const httpProxy = require('@fastify/http-proxy')
 const fp = require('fastify-plugin')
 
 module.exports = fp(async function (app, opts) {
-  for (const { proxy, origin } of opts.services) {
-    if (!proxy) continue
+  for (const service of opts.services) {
+    if (!service.proxy) {
```

Comment:
IMHO use should be able to set `service.proxy = false` to disable this default proxy.

Reply:
Good point, added.